### DISCUSSION TITLE
How do I find a "\uXXXX" glyph that exists in the font I'm using?

### DISCUSSION THREAD

**Discussion by svenlowry-cgi** [JanDeDobbeleer/oh-my-posh#3469] [Troubleshoot] [answered]
I'm currently using CaskaydiaCove Nerd Font Mono for my terminals, and it looks to be missing the Azure Devops Icon (`\uEBE8`) - it just displays a rectangle when I'm working in an ADO repository.

How can I find an alternative in the font to manually set in my config?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@svenlowrybjss you can make use of the nerd font [cheat sheet](https://www.nerdfonts.com/).
